docs/tutorial/dbinit.rst

```diff
@@ -44,6 +44,9 @@ Within the function, you can then access :attr:`flask.g` and other things as
 you might expect.  When the script ends, the application context tears down
 and the database connection is released.
 
+.. note:: In order to use the newly created ``flask initdb`` command, the 
+    application must be re-installed using ``pip install -e .``
```

Comment:
My point is twofold:

* We can have this instruction as normal text instead of a note/alert
* This instruction is not necessarily part of the db initialization. In particular, `pip install -e .` is already required for Step 2 (`flask run`)